jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -85,14 +86,12 @@
 public class DefaultServlet extends HttpServlet
```

Comment:
I would like to see the javadoc restored here asap. As you're editing the class in this PR, I think now is the time to do it.